specification/app/resource-manager/Microsoft.App/preview/2022-06-01-preview/ManagedEnvironments.json

```diff
@@ -753,7 +753,7 @@
               "description": "Azure Monitor instrumentation key used by Dapr to export Service to Service communication telemetry",
               "x-ms-mutability": [
                 "create",
-                "read"
+                "update"
```

Comment:
"x-ms-secret": true is equivalent to "x-ms-mutability": ["create", "update"]. See https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-secret

You can keep one of them